packages/@coorpacademy-components/src/molecule/learning-profile-radar-chart/index.js

```diff
@@ -0,0 +1,395 @@
+import React, {Fragment, useCallback, useEffect, useMemo, useState} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Radar,
+  RadarChart,
+  PolarGrid,
+  PolarAngleAxis,
+  ResponsiveContainer,
+  Tooltip,
+  PolarRadiusAxis
+} from 'recharts';
+import {
+  find,
+  pipe,
+  keyBy,
+  mapValues,
+  size,
+  get,
+  getOr,
+  map,
+  toPairs,
+  values,
+  isEmpty,
+  omit,
+  fromPairs,
+  times,
+  flatten
+} from 'lodash/fp';
+import classnames from 'classnames';
+import {isMobile as getIsMobile} from '../../util/check-is-mobile';
+import style from './style.css';
+
+const uncappedMap = map.convert({cap: false});
+
+// TICK_POSITIONS
```

Comment:
```suggestion
/* TICK_POSITIONS */
```

Utilise plutot le block comment pour délimiter tes sections
(et NE CRIE PAS STEUPLE 😅 )